libnmstate/error.py

```diff
@@ -109,3 +109,11 @@ class NmstateNotSupportedError(NmstateError):
     """
 
     pass
+
+
+class NmstateTimeoutError(NmstateLibnmError):
```

Comment:
The `NmstateLibnmError` is `Exception for unexpected libnm failure.`

But NmstateTimeoutError is expected failure.

Reply:
I am not sure I follow your point here.
An unexpected timeout occurred and instead of raising the general exception, we raise a specific one.